quickstart/fluent-package-v5-vs-td-agent.md

```diff
@@ -0,0 +1,58 @@
+# fluent-package v5 vs td-agent v4
+
+## Supported Platforms
+
+[Fluentd Project](https://www.fluentd.org) maintains stable packages for Fluentd and canonical plugins as Fluent Package \(the package is called `fluent-package`\)
+which is formerly known as `td-agent`.
+ `fluent-package` has v5 and LTS (Long Term Support) version.
+
+| Platform | v4\(x86\_64\) | v4\(Arm64\) | v5\(x86\_64\) | v5\(Arm64\) |
+| :--- | :---: | :---: | :---: | :---: |
+| RedHat/CentOS 7 | ✔ | ✔ | ✔ | ✔ |
+| RedHat/CentOS 8 | ✔ | ✔ | ✔ | ✔ |
+| RedHat/CentOS 9 | ✔ | ✔ | ✔ | ✔ |
+| Amazon Linux 2 |  ✔ | ✔ | ✔ | ✔ |
+| Amazon Linux 2023 | - | - |  ✔ | ✔ |
+| Ubuntu Focal | ✔ | ✔ | ✔ | ✔ |
+| Ubuntu Jammy | ✔ | ✔ | ✔ | ✔ |
+| Debian Buster | ✔ | ✔ | - | - |
+| Debian Bullseye | ✔ | ✔ | ✔ | ✔ |
+| Debian Bookworm | - | - | ✔ | ✔ |
+| macOS | ✔ | - | ✔ | - |
+| Windows | ✔ | - | ✔ |  - |
+
+Basically, `fluent-package` v5 and LTS version supports same platforms.
```

Comment:
Does this mean that the platforms supported by `fluent-package` are basically the same as those by `td-agent`?

Reply:
Or they are basically the same between Normal release and LTS?

Reply:
I mean the latter

Reply:
I see!
Then, I think we don't need to mention this here (as well as https://github.com/fluent/fluentd-docs-gitbook/pull/461#discussion_r1298087646)

It would be more simple and easy to understand not to mention the channels in this section.

Reply:
I got it.